apps/communications/dialer/style/call_log.css

```diff
@@ -192,6 +197,7 @@ ol, ul {
 .log-item .additional-info {
   display: inline-flex;
   justify-content: space-between;
+  margin-top: -0.4rem;
```

Comment:
Don't really like negative margins, but can't think of any better approach to preserve the same look & feel.